### ISSUE TITLE
compile error

### ISSUE THREAD

**Issue by mineralres** [nnethercote/dhat-rs#9]
```
   --> /home/xb/.cargo/registry/src/mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd/dhat-0.2.1/src/lib.rs:483:40
    |
483 | ...                   symbol.colno().unwrap_or(0),
    |                              ^^^^^ method not found in `&BacktraceSymbol`
```

when compile dhat-rs directly there's no error, i can use  'cargo run --example heap' with out any error.
but when i add crate dhat in my project, it's turn out the up front error. 
my os is ubuntu 16.04.

**Comment by nnethercote** [comment]
I saw the same error at one point and fixed it by updating the `backtrace` version in `Cargo.toml` from 0.3 to 0.3.50. What `backtrace` version do you have in your `Cargo.lock` file?

**Comment by lukesteensen** [comment]
I had this same issue and it seems to have been fixed by updating the `backtrace` crate:

```
$ cargo update -p backtrace
    Updating crates.io index
    Updating addr2line v0.13.0 -> v0.14.0
    Updating backtrace v0.3.50 -> v0.3.55
    Updating gimli v0.22.0 -> v0.23.0
      Adding object v0.22.0
```

**Comment by nnethercote** [comment]
Fixed by [2609b1d](https://github.com/nnethercote/dhat-rs/commit/2609b1db6f1163a3b3a8214bca4a35307f62f7e2).
